airbyte-integrations/connectors/source-iterable/source_iterable/api.py

```diff
@@ -158,6 +175,64 @@ def path(self, **kwargs) -> str:
         return "campaigns"
 
 
+class CampaignsMetrics(IterableStream):
+    primary_key = None
+    data_field = None
+
+    def __init__(self, api_key: str, start_date: str):
+        """
+        https://api.iterable.com/api/docs#campaigns_metrics
+        """
+        super().__init__(api_key)
+        self.start_date = start_date
+
+    def path(self, **kwargs) -> str:
+        return "campaigns/metrics"
+
+    def request_params(self, stream_slice: Optional[Mapping[str, Any]] = None, **kwargs) -> MutableMapping[str, Any]:
+        params = super().request_params(**kwargs)
+        params["campaignId"] = stream_slice.get("campaign_id")
+        params["startDateTime"] = self.start_date
+
+        return params
+
+    def stream_slices(self, **kwargs) -> Iterable[Optional[Mapping[str, any]]]:
```

Comment:
How about slice not by one campaign, but by 20-30, since we already doing a lot of sub reads